packages/firebase_in_app_messaging/test/firebase_in_app_messaging_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:firebase_in_app_messaging/firebase_in_app_messaging.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$FirebaseInAppMessaging', () {
+    final List<MethodCall> log = <MethodCall>[];
+
+    setUp(() {
+      log.clear();
+      FirebaseInAppMessaging.channel
+          .setMockMethodCallHandler((MethodCall methodcall) async {
+        log.add(methodcall);
+        return true;
+      });
+    });
+
+    test('triggerEvent', () async {
+      final FirebaseInAppMessaging fiam = FirebaseInAppMessaging();
+      fiam.triggerEvent('someEvent');
```

Comment:
Generally I think we should `await` async methods in tests to ensure success and prevent race conditions.

Reply:
Done